be/src/storage/rowset/array_column_iterator.cpp

```diff
@@ -83,7 +82,11 @@ Status ArrayColumnIterator::next_batch(size_t* n, Column* dst) {
     num_to_read = end_offset - num_to_read;
 
     // 3. Read elements
-    RETURN_IF_ERROR(_element_iterator->next_batch(&num_to_read, array_column->elements_column().get()));
+    if (_access_values) {
+        RETURN_IF_ERROR(_element_iterator->next_batch(&num_to_read, array_column->elements_column().get()));
+    } else {
+        array_column->elements_column()->append_default(num_to_read);
```

Comment:
append_null is also OK?

Reply:
yeah, is same

Reply:
I will remove append-value later, need another PR